src/screens/Profile/FavoriteList/index.tsx

```diff
@@ -54,23 +56,21 @@ export default function FavoriteList({navigation}: ProfileScreenProps) {
               key={favorite.name + favorite.location.id}
               favorite={favorite}
               accessibility={{
-                accessibilityHint: 'Aktivér for å redigere',
+                accessibilityHint: t(FavoriteListTexts.favoriteItem.a11yHint),
               }}
               onPress={navigateToEdit}
             />
           ))}
         </Sections.Section>
 
         <Sections.Section withPadding>
-          {!!items.length && (
```

Comment:
Here we introduce a regression error I think.

Reply:
Somewhat faulty merge with master, probably. Not intentional. I'll look through and compare.